avocado/plugins/docker.py

```diff
@@ -134,11 +141,13 @@ def __init__(self, job, test_result):
     def setup(self):
         dkrcmd = self.job.args.docker_cmd
         dkr_opt = self.job.args.docker_options
-        self.remote = DockerRemoter(dkrcmd, self.job.args.docker, dkr_opt)
+        dkr_name = os.path.basename(self.job.logdir) + '.' + 'avocado'
```

Comment:
We should try to stick with the job ID on cases such as this, but I do get your point about having a hard(er) time to locate the job results with a given job ID.

Until we make that easier (tracking in this card: https://trello.com/c/vrp2F5n4/845-job-id-utility-command-to-find-job-results-by-id) I'm OK with this.